package.json

```diff
@@ -81,6 +81,8 @@
     "karma-spec-reporter": "0.0.32",
     "mocha": "^5.0.1",
     "standard": "^12.0.1",
+    "ts-node": "^7.0.1",
+    "typescript": "^3.1.6",
```

Comment:
Suggestion: If this would be declared as a `peerDependency` users could chose the TS version they prefer. This would prevent us from having to update it ourselves whenever a new TS version comes out.